### PR TITLE
Week 1 MichaelDevelopment

### PR DESCRIPTION
**Pull request by MPKellyy**
This is the pull request that I will be working under for the first week of development (plan to merge everything on Friday).

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/FormHandlerServlet.java
Comment (atsao72) [inline src/main/java/com/google/sps/servlets/FormHandlerServlet.java]: This switch case and the next one look the same, I'd probably create a separate helper function.

private int getScore(char input) {...}

File: src/main/java/com/google/sps/servlets/FormHandlerServlet.java
Comment (atsao72) [inline src/main/java/com/google/sps/servlets/FormHandlerServlet.java]: Usually I'd put the "expectation" in the equals(). so:

request.getParameter("parking").equals("1").

File: src/main/java/com/google/sps/servlets/FormHandlerServlet.java
Comment (atsao72) [inline src/main/java/com/google/sps/servlets/FormHandlerServlet.java]: optional: You could also set parking to result of the boolean. 

parking = "1".equals(...)

File: src/Test.txt
Comment (atsao72) [inline src/Test.txt]: probably delete this file

**Review by atsao72 — COMMENTED** [review]
few minor code cleanup suggestions but functionality looks good

**Review by KeaKing — COMMENTED** [review]
Everything looks good so far, I'll leave it to Adedapo to approve it if he agrees

**Review by Adedapo01 — APPROVED** [review]
Everything looks good!

**Pull request merged**
